Tests/OutlineViewTests/TreeMapTests.swift

```diff
@@ -0,0 +1,139 @@
+import XCTest
+@testable import OutlineView
+
+final class TreeMapTests: XCTestCase {
+    struct TestItem: Identifiable, Equatable {
+        var id: Int
+        var children: [TestItem]?
+    }
+
+    /// A basic TreeMap with 5 root objects (1 through 5),
+    /// where all but item 5 are internal nodes, and all
+    /// are collapsed.
+    static var TestTree: TreeMap<Int> {
+        let rootItems = (1...5)
+            .map { TestItem(id: $0, children: $0 == 5 ? nil : []) }
+            .map { OutlineViewItem(value: $0, children: \TestItem.children) }
```

Comment:
Are you able to elide the type name here?

```suggestion
            .map { OutlineViewItem(value: $0, children: \.children) }
```

Reply:
I just tried, and it looks like the compiler isn't happy with removing the type name. It was worth a shot :-)